### DISCUSSION TITLE
2.0.3通过hms可以查看到hudi表，但查询不到数据

### DISCUSSION THREAD

**Discussion by juice411** [apache/doris#30569] [A - General / Q&A]
通过hms可以获取到hive元数据，但查询不到hudi表数据，有人遇到类似问题码？

**Comment by LemonLiTree** [comment]
报错是啥

**Comment by juice411** [comment]
你好，相关问题已和你们的技术支持在微信群得以解释，查不到数据有几个方面：
1、我们的hms服务是可以看得到hudi表、hudi同步表
2、同理Doris通过hms也是可以看到所有的表
3、但hive、doris是不能直接查询hudi原表的，只能查询对应的同步表，这点我们忽略了，以为可以直接查，所以查询出问题
4、Doris查询对应的hudi同步表，同步表有cow和mor两种，mor如果没有进行合并的话，hive是查不到数据的，同理Doris也查不到
基本就是这些问题，现在也弄明白了，感谢你们的技术支持。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/doris"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月1日(星期四) 下午3:38
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/doris] 2.0.3通过hms可以查看到hudi表，但查询不到数据 (Discussion #30569)





 
报错是啥
